src/components/dropdownSearchbar/dropdownSearchbar.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import Select from 'react-select'
+import './dropdownSearchbar.css'
+
+function DropdownSearchbar() {
+  const [selectedOption, setSelectedOption] = useState('Select Patient Group')
+
+  const aquaticCreatures = [
+    { label: 'Shark', value: 'Shark' },
+    { label: 'Dolphin', value: 'Dolphin' },
+    { label: 'Whale', value: 'Whale' },
+    { label: 'Octopus', value: 'Octopus' },
+    { label: 'Crab', value: 'Crab' },
+    { label: 'Lobster', value: 'Lobster' },
+  ]
+
+  const handleTypeSelect = (e) => {
+    setSelectedOption(e.value)
+  }
+
+  return (
+    <div className="SearchBar">
+      <div
+        style={{
+          boxSizing: 'border-box',
+          width: '100%',
+          justifyContent: 'center',
+          backgroundColor: 'rgb(229, 229, 229)',
+          display: 'flex',
+          padding: '1px',
+        }}
```

Comment:
Let's move this style to a separate class.

Reply:
created issues when moving it to a css file and based on timeboxing this issue i feel out of time.